src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverse = {};
+
+  for (const i in robot) {
```

Comment:
Let's create meaningful names for variables
```suggestion
  for (const key in robot) {
```